packages/component/src/createLoadable.js

```diff
@@ -151,16 +151,22 @@ function createLoadable({
           if (options.ssr === false) {
             return
           }
+          
+          if (!options.suspense) {
```

Comment:
this is not quite about Suspense, but also about a particular way you render application. For any "old" React API this change will lead to an exception.

We cannot do it that simply. Some external configuration is required.

Reply:
I  not sure this is a issue since it is inside if chunkextractor on the server and lazy never worked on the server.